System-Tests/tests/nightwatch/dashboard/dashboard_device_widget.spec.js

```diff
@@ -83,6 +84,27 @@ module.exports = {
             throw new Error(error);
         }
     },
+    "Download and verify backup": async function (browser) {
```

Comment:
`"Download and verify backup"` -> "Download and verify that backup is a simple JSON string"